README.md

```diff
@@ -285,58 +272,62 @@ AutoGen 0.2 can be installed with:
 
 ### Is 0.4 done?
 
-We are still actively developing AutoGen 0.4. One exciting new feature is the emergence of new SDKs for .NET. The python SDKs are further ahead at this time but our goal is to achieve parity. We aim to add additional languages in future releases.
+We are still actively developing AutoGen 0.4. One exciting new feature is the emergence of new SDKs for .NET. 
+The Python SDKs are further ahead at this time, but our goal is to achieve parity. We aim to add additional languages in future releases.
 
 ### What is happening next? When will this release be ready?
 
-We are still working on improving the documentation, samples, and enhancing the code. We are hoping to release before the end of the year when things are ready.
+We are still working on improving the documentation, samples, and enhancing the code. 
+We hope to release before the end of the year when things are ready.
 
 ### What is the history of this project?
 
-The rearchitecture of the framework started with multiple Microsoft teams coming together
-to address the gaps and learnings from AutoGen 0.2 - merging ideas from several predecessor projects.
+The rearchitecture of the framework started with multiple Microsoft teams coming together to address the gaps and learnings from AutoGen 0.2, 
+merging ideas from several predecessor projects. 
 The team worked on this internally for some time to ensure alignment before moving work back to the open in October 2024.
 
 ### What is the official channel for support?
 
-Use GitHub [Issues](https://github.com/microsoft/autogen/issues) for bug reports and feature requests.
+Use GitHub [Issues](https://github.com/microsoft/autogen/issues) for bug reports and feature requests. 
 Use GitHub [Discussions](https://github.com/microsoft/autogen/discussions) for general questions and discussions.
 
 ### Do you use Discord for communications?
 
-We are unable to use Discord for project discussions. Therefore, we request that all discussions take place on <https://github.com/microsoft/autogen/discussions/> going forward.
+We are unable to use Discord for project discussions. 
+Therefore, we request that all discussions take place on <https://github.com/microsoft/autogen/discussions/> going forward.
 
 ### What about forks?
 
-<https://github.com/microsoft/autogen/> remains the only official repo for development and support of AutoGen.
-We are aware that there are thousands of forks of AutoGen, including many for personal development and startups building with or on top of the library. We are not involved with any of these forks and are not aware of any plans related to them.
+https://github.com/microsoft/autogen/ remains the only official repo for development and support of AutoGen.
+We are aware that there are thousands of forks of AutoGen, including many for personal development and startups building with or on top of the library. 
+We are not involved with any of these forks and are not aware of any plans related to them.
```

Comment:
Can we not have these new line edits (this and many others in this PR) if there is not edit to the context. Thanks!